atd-vzd/migrations/default/1725561923206_cris_v28_lookup_changes/up.sql

```diff
@@ -0,0 +1,105 @@
+delete from lookups.agency where id = 2092;
+delete from lookups.agency where id = 212;
+delete from lookups.agency where id = 2202;
+delete from lookups.agency where id = 220;
+delete from lookups.agency where id = 2272;
+delete from lookups.agency where id = 2296;
+delete from lookups.agency where id = 2314;
+delete from lookups.agency where id = 2320;
+delete from lookups.agency where id = 234;
+delete from lookups.agency where id = 23673;
+delete from lookups.agency where id = 23682;
+delete from lookups.agency where id = 23773;
+delete from lookups.agency where id = 23794;
+delete from lookups.agency where id = 23816;
+delete from lookups.agency where id = 23845;
+delete from lookups.agency where id = 23846;
+delete from lookups.agency where id = 23854;
+delete from lookups.agency where id = 23897;
+delete from lookups.agency where id = 24043;
+delete from lookups.agency where id = 2446;
+delete from lookups.agency where id = 2461;
+delete from lookups.agency where id = 2586;
+delete from lookups.agency where id = 2590;
+delete from lookups.agency where id = 2602;
+delete from lookups.agency where id = 262;
+delete from lookups.agency where id = 275;
+delete from lookups.agency where id = 2779;
+delete from lookups.agency where id = 2831;
+delete from lookups.agency where id = 2839;
+delete from lookups.agency where id = 2871;
+delete from lookups.agency where id = 2872;
+delete from lookups.agency where id = 2887;
+delete from lookups.agency where id = 3078;
+delete from lookups.agency where id = 3123;
+delete from lookups.agency where id = 3168;
+delete from lookups.agency where id = 3171;
+delete from lookups.agency where id = 3173;
+delete from lookups.agency where id = 3174;
+delete from lookups.agency where id = 3181;
+delete from lookups.agency where id = 3183;
+delete from lookups.agency where id = 3186;
+delete from lookups.agency where id = 321;
+delete from lookups.agency where id = 3250;
+delete from lookups.agency where id = 3254;
+delete from lookups.agency where id = 3257;
+delete from lookups.agency where id = 3258;
+delete from lookups.agency where id = 3262;
+delete from lookups.agency where id = 3264;
+delete from lookups.agency where id = 3270;
+delete from lookups.agency where id = 3273;
+delete from lookups.agency where id = 3275;
+delete from lookups.agency where id = 3281;
+delete from lookups.agency where id = 3283;
+delete from lookups.agency where id = 3284;
+delete from lookups.agency where id = 3287;
+delete from lookups.agency where id = 3290;
+delete from lookups.agency where id = 3291;
+delete from lookups.agency where id = 3295;
+delete from lookups.agency where id = 3299;
+delete from lookups.agency where id = 3310;
+delete from lookups.agency where id = 3313;
+delete from lookups.agency where id = 3314;
+delete from lookups.agency where id = 3323;
+delete from lookups.agency where id = 3335;
+delete from lookups.agency where id = 42;
+delete from lookups.agency where id = 71;
```

Comment:
The crashes with `crashes.id` of the following use this `lookups.agency` record and the migration fails when run on today's replicated data.


```
334511, 337719, 340347, 344033, 345331, 347562, 356144, 401431, 404437, 404442
```

Reply:
i saw the same error when running the up migration after replicating data today.

Reply:
yes! the migration failed by design, but the conflict is surprise to me 😞  thank you both for testing. i'm looking to this.